tests/test_availability_filter.py

```diff
@@ -0,0 +1,9 @@
+from availability_filter import AvailabilityFilter
+from beer import Beer
+from yes_filter import YesFilter
+
+
+def test_name_filter_match():
```

Comment:
You also need a test for not available.  And given my discussion in the filter class, you will need negative quantities.